src/main/kotlin/no/nav/syfo/Environment.kt

```diff
@@ -12,7 +12,7 @@ data class Environment(
     val inputQueueName: String = getEnvVar("MQ_INPUT_QUEUE_NAME"),
     val apprecQueueName: String = getEnvVar("MQ_APPREC_QUEUE_NAME"),
     val aktoerregisterV1Url: String = getEnvVar("AKTOR_REGISTER_V1_URL"),
-    val kuhrSarApiUrl: String = getEnvVar("KUHR_SAR_API_URL", "http://kuhr-sar-api"),
+    val kuhrSarApiUrl: String = getEnvVar("KUHR_SAR_API_URL", "http://kuhr-sar-api.teamkuhr.svc.nais.local"),
```

Comment:
Vil `nais.local` fungere i prod?

Reply:
den skal visstnok det

Reply:
Det er sånn de gjør det i team sykmelding for alle sine apper som bruker kuhr sar